src/OpenDiffix.Core/Utils.fs

```diff
@@ -69,3 +69,8 @@ module Hash =
 
   let strings start data =
     data |> Seq.distinct |> Seq.map string |> Seq.fold (^^^) start
+
+// introduced for compatibility of math-related functions with C (PostgreSQL)
+module CMath =
```

Comment:
I would rename this module to `Math` (F# allows module merging) and the function to `roundAwayFromZero` or  something similar.

Reply:
Sounds good. This has a slight drawback of being longer and chopping up some lines in `Expressions.fs` (even if we `open` it) but :shrug: